### ISSUE TITLE
Link to another note in the same book opens in a new tab in Chrome.

### ISSUE THREAD

**Issue by benabraham** [hackmdio/hackmd-io-issues#170]
### Is this a hackmd.io issue?

- Yes, the bug happens on https://hackmd.io

#### Current behaviour

1. Go to https://hackmd.io/@benabraham/book-example/
2. Open any link on the right side: link in “Go to Page B” etc.
3. Link opens in a new tab (most of the time).

![video15](https://user-images.githubusercontent.com/334494/77911771-ac9cc100-7291-11ea-898c-f1e0bf0d9330.gif)

###### Sources

- Book https://hackmd.io/0HWwfBvSR_eeA4f95qteZA
- Page A: https://hackmd.io/bC65cLJtT4iRtmCLxbIPKg
- Page B: https://hackmd.io/qoN4Ee9NSY-r9Adv4avV3g
- Page C: https://hackmd.io/uq8RHC3gQfWAb4QvqoUakQ

#### Expected behaviour

**I expect note to open in the same tab and in the right pane of the book**: the same it works in both the latest Firefox and Edge 18 because links have exactly the same URL as in the „book“.

#### Environment
I use HackMD on **Desktop**

- OS: Windows 10
- Browser: Chrome 64-bit
- Browser Version: 80.0.3987.149

This behavior is also visible in:

- OS:  Linux Mint 19.3
- Browser: Chromium
- Browser Version: 80

and mobile

- OS:  Android 9
- Browser: Chrome
- Browser Version: 80

#### Additional context

I tested it using a clean user profile (everything is default, no extensions except Google's default for offline Docs use).

**Comment by reagle** [comment]
I suspect this behavior (always open links in a new tab, except links in a book frame) is the default/expected. See "Does clicking on a link always require a new tab?" #110 .

**Comment by benabraham** [comment]
It should open exactly the same links inside the right pane: see https://github.com/hackmdio/codimd/issues/658

If not, it would be weird that it works in every other browser except Chrome.
And in Chrome it seems to work randomly from time to time…

**Comment by reagle** [comment]
You are correct, no no tabs on Firefox. My theory at the time is that Firefox ignores _target directives, but I don't know...

**Comment by benabraham** [comment]
No, it works. Check yourself. https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/target

To me it seems that the JavaScript that changes that behavior has a bug in Chrome.
